src/components/sendAskMoney/SendAskMoney.vue

```diff
@@ -608,6 +608,9 @@
 
       async newLinkTab() {
         if (this.amountForCredit > 0) {
+        // This to ensure we are left with 2 decimals only
+        this.amountForCredit =
+          Math.round((this.amountForCredit + Number.EPSILON) * 100) / 100
```

Comment:
You are right in that we do not need that huge of an amount of precision since we are dealing with an amount input, and for a currency it does not make sense to have more than two digits entered by the user. We only do the rounding to ensure no mistake is accidentally made...
Moreover, `Number.EPSILON` is not set in InternetExplorer so  wee should keep maximum backwards compatibility.

There was also another line of code including this formula which I also changed.

So thanks for feedback. Updated the commits.